### ISSUE TITLE
Text classification with movie reviews lacks examples of results at end

### ISSUE THREAD

**Issue by rmaunder** [tensorflow/tensorflow#27792]
The "[Text classification with movie reviews](https://www.tensorflow.org/tutorials/keras/basic_text_classification)" :

could really do with some examples of correct / incorrect classification as you work through / at end. At present the dataset feels a bit like a black box, and a user doesn't really get a feel for the claimed 87% accuracy (c.f. the earlier image classification where we see samples of correct/incorrect labelling).

As example the sanity check found at end of this similar example: https://towardsdatascience.com/sentiment-analysis-with-python-part-1-5ce197074184
with 5 most discriminating words for both positive and negative reviews would give more of a feel to the user on how the model was performing?

**Comment by gadagashwini** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged//tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.
Thanks!

**Comment by gadagashwini** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
